spec/lib/tasks/delete_old_blackouts_rake_spec.rb

```diff
@@ -4,30 +4,27 @@
 describe 'delete_old_blackouts' do
   include_context 'rake'
 
-  before(:all) { FactoryGirl.create(:app_config) }
-
   before(:each) do
+    @ac = mock_app_config
+    allow(AppConfig).to receive(:blank?) { false }
```

Comment:
It's funny how we used different checks in different rake tasks (i.e. checking the [instance](https://github.com/YaleSTC/reservations/pull/1566/files#diff-6a4dd2960a906529c4d4662fe8499b2aR8) vs the class here).